Testing/integration/test_sync_changes.sh

```diff
@@ -25,6 +25,11 @@ if [[ "$(sudo santactl status --json | jq .daemon.block_usb)" != "false" ]]; the
   exit 1
 fi
 
+# Wait for the UI to have come up
+sleep 5
```

Comment:
Can we wait for the distributed notification instead of sleeping?

I believe you can use `notifyutil(1)`   to wait for the notification.

Reply:
Sorry I got it confused with [dndump](https://github.com/nktzbkv/dndump/tree/main)

Reply:
We could, however there's still some delay between the notification being received and the UI coming up, and since the UI is the bit we actually care about here, we're back to square one.

Reply:
Yeah I guess the thought would be that you could wait for the block then only sleep for 1 second as you know the dialog will pop up.

We can look into this later.